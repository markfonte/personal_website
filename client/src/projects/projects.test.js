import React from 'react';
import Projects from './projects.jsx';
import renderer from 'react-test-renderer';

test('Projects matches snapshot', () => {
  const component = renderer.create(
      <Projects/>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
