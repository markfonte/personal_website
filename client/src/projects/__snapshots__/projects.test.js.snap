// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects matches snapshot 1`] = `
<div>
  <p
    className="MuiTypography-root Projects-title-1 MuiTypography-h6 MuiTypography-paragraph"
  >
    My work experiences, internships, side projects and hackathon projects
  </p>
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-colorSecondary"
  >
    <b>
      Click to expand/collapse cards
    </b>
  </p>
  <div
    className="MuiCircularProgress-root MuiCircularProgress-colorSecondary MuiCircularProgress-indeterminate"
    role="progressbar"
    style={
      Object {
        "height": 40,
        "width": 40,
      }
    }
  >
    <svg
      className="MuiCircularProgress-svg"
      viewBox="22 22 44 44"
    >
      <circle
        className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
        cx={44}
        cy={44}
        fill="none"
        r={20.2}
        strokeWidth={3.6}
        style={Object {}}
      />
    </svg>
  </div>
</div>
`;
